rasa/core/tracker_store.py

```diff
@@ -64,6 +65,19 @@
 DEFAULT_REDIS_TRACKER_STORE_KEY_PREFIX = "tracker:"
 
 
+def check_if_tracker_store_async(tracker_store: "TrackerStore") -> bool:
```

Comment:
```suggestion
def check_if_tracker_store_async(tracker_store: TrackerStore) -> bool:
```

(you can add `from __future__ import annotations` to the top of the file if this is not working)